packages/protobuf/src/messages-schema.ts

```diff
@@ -1694,6 +1695,7 @@ export enum Enum_Capability {
     Capability_ShamirGroups = 16,
     Capability_PassphraseEntry = 17,
     Capability_Solana = 18,
+    Capability_Translations = 19,
```

Comment:
this will likely require updating connect unit test. my guess. the test is not the most maintainable one, I agree